### DISCUSSION TITLE
Pycaret Feature Importance Plot

### DISCUSSION THREAD

**Discussion by KingKai69** [pycaret/pycaret#3950] [General]
Hey community,

i am using the pycaret feature importance plot and i was wondering how the feature importance is calculated?

plot_model(model, plot='feature')

I cannot find any information about the calculation in the docs. Any help is appreciated.

Best regards
Kai

**Comment by pascal456** [comment]
I was wondering about this as well. I browsed the code and could not exactly figure this out. I think the corresponding code must be this (Tag / Version `3.3.1`):

https://github.com/pycaret/pycaret/blob/73fe135f9e49bdaabf8bdcb06b3b174000ff35b0/pycaret/internal/pycaret_experiment/tabular_experiment.py#L1815-L1871
